docker/k8s/vtadmin/Dockerfile

```diff
@@ -21,7 +21,6 @@ FROM node:16-${DEBIAN_VER} as node
 
 # Set up Vitess environment (just enough to run pre-built Go binaries)
 ENV VTROOT /vt
-ENV VTADMIN_WEB_PORT=14201
```

Comment:
I think we still need this to configure the port used by vtadmin web (nginx in your PR)